composer.json

```diff
@@ -1,50 +1,34 @@
 {
     "name": "sylake/sylakim",
-    "type": "symfony-bundle",
-    "description": "Connector between Akeneo PIM and Sylius",
-    "keywords": ["pim", "akeneo", "ecommerce", "e-commerce", "sylius", "connector"],
+    "description": "",
+    "type": "akeneo-connector",
     "license": "MIT",
     "authors": [
         {
-            "name": "Julien Janvier",
-            "email": "jjanvier@gmail.com"
-        },
-        {
-            "name": "Arnaud Langlade",
-            "email": "arn0d.dev@gmail.com"
-        },
-        {
-            "name": "Romain Monceau",
-            "email": "monceau.romain@gmail.com"
-        }
-    ],
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/akeneo/pim-community-dev.git"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/akeneo/platform.git"
+            "name": "Kamil Kokot",
+            "email": "kamil@kokot.me",
+            "homepage": "http://kamil.kokot.me"
         }
     ],
-    "minimum-stability": "dev",
     "require": {
-        "akeneo/pim-community-dev": "~1.2",
-        "guzzlehttp/guzzle":        "~3.8"
+        "akeneo/pim-community-dev": "^1.7",
+        "symfony/symfony": "^2.7"
```

Comment:
`symfony/symfony` is already a PIM dependency. Do you really need to add it here?